resources/helm/overlays/istio-telemetry/kiali/templates/kiali-cr.yaml

```diff
@@ -46,21 +46,25 @@ spec:
     {{- end }}
 {{- end }}
 
-{{- if or .Values.kiali.tolerations .Values.kiali.deployment_tolerations }}
+{{- if or .Values.kiali.tolerations .Values.kiali.deployment_tolerations .Values.global.defaultTolerations }}
     tolerations:
     {{- if .Values.kiali.tolerations }}
 {{ toYaml .Values.kiali.tolerations | indent 6 }}
-    {{- else }}
+    {{- else if .Values.kiali.deployment_tolerations }}
 {{ toYaml .Values.kiali.deployment_tolerations | indent 6 }}
+    {{- else }}
+{{ toYaml .Values.global.defaultTolerations | indent 6 }}
```

Comment:
do we need an `if .Values.global.defaultTolerations` here?

Reply:
No, since we already check this in the outer if statement.

Reply:
🤦